msal/src/main/java/com/microsoft/identity/client/internal/CreateApplicationResult.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.identity.client.internal;
+
+import com.microsoft.identity.client.IPublicClientApplication;
+import com.microsoft.identity.client.exception.MsalException;
+
+public class CreateApplicationResult  {
+
+    private MsalException mMsalException;
+    private IPublicClientApplication mIPublicClientApplication;
+    private Boolean mSuccess = false;
+
+    public CreateApplicationResult(IPublicClientApplication application, MsalException exception){
```

Comment:
Could apply `@NonNull`, `@Nullable` annots - not a functional change

Reply:
fixed.

Reply:
fixed.

Reply:
+1, also `final`